modules/CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ endif()
 add_subdirectory( mod-script-pipe )
 
 
-if( NOT CMAKE_SYSTEM_NAME MATCHES "Darwin" )
+if( NOT CMAKE_SYSTEM_NAME MATCHES "Darwin|Windows" )
    if( NOT "${CMAKE_GENERATOR}" MATCHES "Visual Studio*")
```

Comment:
Do we need the check for CMAKE_GENERATOR here? Looks like this condition is always true since we are not on Windows as per the previous clausese

Reply:
@Paul-Licameli what do you think?

Reply:
So if the inner condition is false, you ARE using Visual Studio, but cross-compiling to a non-Windows and non-Darwin system.  Is that possible?

Reply:
At this point in time Visual Studio only runs on Windows and for Windows, unless something has changed recently.
Visual Studio Code (which is cross-platform) is completely different from Visual Studio, as you likely already know

Reply:
That is exactly my point. Please note, that the check is against `CMAKE_GENERATOR`. There is no generator for VSCode, so as you have added `|Windows` - the second check can be removed.

Reply:
Should I remove that check for Visual Studio then?